lucet-validate/Cargo.toml

```diff
@@ -39,6 +39,6 @@ assets = [
     ["LICENSE", "/opt/fst-lucet-validate/share/doc/lucet-validate/", "644"],
     ["../wasi/phases/unstable/witx/typenames.witx",
      "/opt/fst-lucet-validate/share/wasi/unstable/typenames.witx", "644"],
-    ["../wasi/phases/unstable/witx/wasi_unstable_preview0.witx",
+    ["../wasi/phases/unstable/witx/wasi_unstable.witx",
      "/opt/fst-lucet-validate/share/wasi/unstable/wasi_unstable_preview0.witx", "644"],
```

Comment:
Do we also want to change the destination filename?

Reply:
Ah, good catch

Reply:
These paths are wrong, we apparently never make the debs in CI. I'll add it to the makefile.